experiment/src/org/labkey/experiment/pipeline/MoveRunsTask.java

```diff
@@ -220,6 +220,15 @@ public Path getRootPath()
             return FileUtil.stringToPath(_sourceContainer, _root);
         }
 
+        @Override
+        public Path getJobRootPath()
+        {
+            var pipelineJob = getXarContext().getJob();
+            return pipelineJob != null
+                    ? FileUtil.stringToPath(getXarContext().getContainer(), pipelineJob.getPipeRoot().getRootFileLike().toNioPathForRead().toString())

```

Comment:
Do we need to go through `stringToPath`?

```suggestion
                    ? pipelineJob.getPipeRoot().getRootFileLike().toNioPathForRead()

```

Reply:
I thought that was needed to handle cloud-scheme paths.  But `FileUtil.getAbsolutePath(_xarSource.getJobRootPath())` in `XarReader `should take care of that.  I've committed the changes.